index.css

```diff
@@ -1,3 +1,18 @@
-html {
-  
+html {}
+.board { margin:1em auto; border-collapse:collapse }
```

Comment:
I'd recommend keeping your formatting consistent, so I'd write this on multiple lines like you do for your other formatting.

Additionally, and it's not really worth your time unless you really care about it or your team prefers to do it this way, but some people recommend putting the attributes in alphabetical order when possible. Personally, I didn't like to do that on personal projects, and instead grouped them in a way that made sense to me (borders and margins together, width/height together, etc).